actions/workflow.jupyter.user.setup.yaml

```diff
@@ -0,0 +1,27 @@
+name: workflow.jupyter.user.setup
```

Comment:
Nit: could we call it `jupyter.user.prepare`, as this prepares the platform for a course
Setup is more tweaking the configuration in my mind.

This is a really pedantic nit even for me :sweat_smile: